src/main/java/core/basesyntax/strategy/operation/ReturnOperation.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax.strategy.operation;
+
+import core.basesyntax.dao.StorageDao;
+import core.basesyntax.dao.StorageDaoImpl;
+
+public class ReturnOperation implements OperationHandler {
+    private StorageDao storageDao = new StorageDaoImpl();
```

Comment:
The `ReturnOperation` class directly instantiates `StorageDaoImpl`. This tightly couples the class to a specific implementation of `StorageDao`. Consider using dependency injection to pass in the `StorageDao` instance, which will make the class more flexible and easier to test.